assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -80,6 +80,147 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	describe( "should render an open text question when the `question_type` is 'open_text'", () => {
+		beforeEach( () => {
+			registry
+				.dispatch( CORE_USER )
+				.receiveTriggerSurvey( fixtures.singleQuestionOpenText, {
+					triggerID: 'jestSurvey',
+				} );
+
+			fetchMock.post(
+				/^\/google-site-kit\/v1\/core\/user\/data\/survey-event/,
+				{ body: {}, status: 200 }
+			);
+		} );
+
+		it( 'should limit text input to 100 characters', async () => {
+			const {
+				getByText,
+				getByPlaceholderText,
+				getByDisplayValue,
+			} = render( <CurrentSurvey />, {
+				registry,
+			} );
+
+			// Check question_text is set by question_text prop.
+			expect(
+				getByText( 'How satisfied are you with Site Kit?' )
+			).toBeInTheDocument();
+
+			// Check subtitle is set by subtitle prop.
+			expect(
+				getByText( 'Based on your experience so far, tell us.' )
+			).toBeInTheDocument();
```

Comment:
These tests are much better separated, thanks!

These two expect statements in particular aren't really related to the 100 character limit tests, so could also be separated. I think that'd be good, but these two (`question_text` and `subtitle` prop) could probably be in the same `it()` block.

Reply:
Understood. I just didn't want to keep adding extra small tests but I'm sure these are quick to run anyway :+1:

Reply:
https://kentcdodds.com/blog/write-fewer-longer-tests

Initially I was following the react-testing-library creator's advice, but it's not a hard science :smile: